code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -124,3 +124,15 @@
 		icon_state = "biofoam1"
 	else
 		icon_state = "biofoam0"
+
+/obj/item/weapon/reagent_containers/hypospray/autoinjector/paralytic_injector
+	name = "paralytic injector"
+	desc = "A small, single-use device used to administer small amounts of paralytic agent."
+	amount_per_transfer_from_this = 15
+	volume = 15
+	flags = FPRINT
+
+/obj/item/weapon/reagent_containers/hypospray/autoinjector/paralytic_injector/New()
+	..()
+	reagents.remove_reagent(DOCTORSDELIGHT, 30)
+	reagents.add_reagent(AMASEC, 1)
```

Comment:
I like the effect Amasec has, but it's a bardrink. It would be nice if there was an actual chem that behaved similarly to Amasec that they could mix up in chemistry. Since you have a little time to work on this PR (as American Lights has mapping priority), I suggest you make a new chem, which has a few advantages.
* Mixable in chemistry.
* Better immersion instead of being an Imperium bar drink.
* Amasec can stack up to really high paralysis if they get injected with a lot because of AdjustStunned. Instead, if we use SetStunned, they will stop being stunned quickly after they are purged of the chemical.
* By adding an overdose we can discourage orderlies from putting a lot in people, in the same way chloral has to be measured if you aren't trying to kill someone.

```
/datum/reagent/suxameth
	name = "Suxameth"
	id = SUX
	description = "A name for Suxamethonium chloride. A medical full-body paralytic preferred because it is easy to purge."
	reagent_state = REAGENT_STATE_LIQUID
	color = "#664300" //rgb: 102, 67, 0
	overdose_am = 11

/datum/reagent/suxameth/on_mob_life(var/mob/living/M)
	if(..())
		return 1

	M.SetStunned(2)

/datum/reagent/suxameth(var/mob/living/M)
	M.adjustOxyLoss(4) //Very rapid deterioration if they go over 10 units
```

with a recipe
(Toxin is easily obtained from a nanomed vendor but not easily found in most other places, which means this will be harder to ghettochem -- you can still do it if you're clever though)

```
/datum/chemical_reaction/suxameth
	name = "Suxameth"
	id = SUX
	result = SUX
	required_reagents = list(TOXIN = 1, CHLORINE = 1)
	result_amount = 2
```

I also recommend changing the injectors to contain 5 units, so it lasts 10 ticks with 1 default stab, 2 stabs is safe, but 3 stabs will overdose.

Reply:
I considered adding a new reagent earlier considering the overall lack of a proper paralytic agent that wouldn't be overwhelmingly broken, but I was concerned about going over the top. I like the look of the suxameth suggestion here. I'll go ahead and add it.